source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -123,11 +123,12 @@ content: |
 
         .. code-block:: java
            :emphasize-lines: 2
-
+           
```

Comment:
Let's remove extra trailing whitespace. I think VSCode has an option to automatically do that.

Reply:
Hey, not sure what you mean by extra trailing-whitespace here (unless the github comment is pointing to the wrong line)